lib/run_cucumber.js

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env node
+
+const fs = require('fs');
+const log = require('./utils/log');
+const session_vars = require('./utils/session_vars');
+
+// import {loadConfiguration, loadSupport, runCucumber } from '@cucumber/cucumber/api';
+const cucumber_api = require('@cucumber/cucumber/api');
+
+(async function main() {
+  let argv = process.argv;
+
+  let just_args = argv.slice(2);
+
+  const environment = { cwd: process.cwd() };
+  const provided_config = {
+    "publishQuiet": true,
+    "require": "./lib/index.js", 
+    "format": ["progress", "summary:cucumber-report.txt"],
```

Comment:
Do we need these if they're in cucumber.js? Where is cucumber.js being used?

Reply:
Cucumber.js isn't being used anymore, we should be able to remove